pyanaconda/modules/storage/partitioning/interactive_utils.py

```diff
@@ -455,3 +455,81 @@ def collect_device_types(device, disks):
         supported_types.add(devicefactory.DEVICE_TYPE_BTRFS)
 
     return sorted(filter(devicefactory.is_supported_device_type, supported_types))
+
+
+def add_device(storage, dev_info):
+    """Add d device to the storage model.
```

Comment:
A typo

Reply:
Fixed in https://github.com/rhinstaller/anaconda/pull/2040/commits/a4406bd1aa9316e7c3440a3c0aa17a5567fd9214.